### PR TITLE
LookML updates; built out PDT for Asset Downloads

### PR DESCRIPTION
**Pull request by BJenning**

### CODE REVIEW

File: asset_downloads.view.lkml [hunk 1]
```diff
@@ -1,56 +1,107 @@
...
+    type:  string
+    sql:  ${TABLE}.referrer ;;
+    group_label: "Referrer"
   }
 
-  dimension:  asset_path {
+  dimension: referrer_urlhost {
     type:  string
-    sql: 'https://www2.gov.bc.ca' || replace(replace(${TABLE}.request_string,'GET ',''), ' HTTP/1.0','') ;;
+    sql:  ${TABLE}.referrer_urlhost ;;
```
Comment (mark-walle) [inline asset_downloads.view.lkml]: I don't see this column in the table above. Trying to explore it I got this error:
```
ERROR: column asset_downloads.referrer_urlhost does not exist
```

File: asset_downloads.view.lkml [hunk 2]
```diff
@@ -1,56 +1,107 @@
...
+    timeframes: [hour,date,week,month,quarter,year]
+    sql: ${TABLE}.date_timestamp ;;
+    group_label: "Download"
   }
 
-  dimension: ip {
+  dimension: ip_address {
     type: string
-    sql: ${TABLE}.ip ;;
+    sql: ${TABLE}.ip_address ;;
```
[the remark this answers is not in the document]
Comment (BJenning) [reply asset_downloads.view.lkml]: Added description

File: asset_downloads.view.lkml [hunk 3]
```diff
@@ -1,56 +1,107 @@
...
     type: number
     sql: ${TABLE}.proxy ;;
   }
 
-  dimension: referrer {
-    type: string
-    sql: ${TABLE}.referrer ;;
+  dimension: status_code {
+    type: number
+    sql: ${TABLE}.status_code ;;
```
[the remark this answers is not in the document]
Comment (BJenning) [reply asset_downloads.view.lkml]: Added description

File: asset_downloads.view.lkml [hunk 4]
```diff
@@ -1,56 +1,107 @@
...
     type: string
-    sql: ${TABLE}.request_string ;;
+    sql: ${TABLE}.user_agent_http_request_header ;;
+    description: "The request line in quotes, composed of the request method, the url, and the HTTP version"
   }
 
-  dimension: return_size {
-    type: number
-    sql: ${TABLE}.return_size ;;
+  dimension: is_efficiencybc_dev {
```
[the remark this answers is not in the document]
Comment (BJenning) [reply asset_downloads.view.lkml]: Thats what its called in the clicks table. I was told to replicate what is in that table.

File: asset_downloads.view.lkml [hunk 5]
```diff
@@ -1,56 +1,107 @@
...
+    type: yesno
+    sql:  ${TABLE}.is_efficiencybc_dev ;;
   }
 
-  dimension: status_code {
-    type: number
-    sql: ${TABLE}.status_code ;;
+  dimension: is_government {
```
[the remark this answers is not in the document]
Comment (BJenning) [reply asset_downloads.view.lkml]: Added description

File: asset_downloads.view.lkml [hunk 6]
```diff
@@ -1,56 +1,107 @@
...
 
-  dimension:  asset_path {
+  dimension: referrer_urlhost {
     type:  string
-    sql: 'https://www2.gov.bc.ca' || replace(replace(${TABLE}.request_string,'GET ',''), ' HTTP/1.0','') ;;
+    sql:  ${TABLE}.referrer_urlhost ;;
+    group_label: "Referrer"
+  }
+
+  dimension: is_mobile {
```
[the remark this answers is not in the document]
Comment (BJenning) [reply asset_downloads.view.lkml]: Added description

File: asset_downloads.view.lkml (hunk 1 above)
Comment (BJenning) [reply asset_downloads.view.lkml]: Im not able to reproduce this error. Can you pull from remote and try again?

Comment (BJenning) [reply asset_downloads.view.lkml]: Resolved in commit ef17f84f403aba4d893158594775da1eaf265062.

**Comment by BJenning** [comment]
I need to run the microservice on all the uploads before I can merge this.

**Pull request merged**